creator-node/src/services/sync/secondarySyncFromPrimary.js

```diff
@@ -90,6 +91,12 @@ const handleSyncFromPrimary = async ({
         }
         throw returnValue.error
       }
+
+      if (forceWipe) {
```

Comment:
yeah def want to be able to have metric for forceWipe or `MergePrimaryAndWipeSecondary` one way or another